src/cortxfs/cortxfs_fops.c

```diff
@@ -184,21 +184,16 @@ static inline ssize_t __cfs_write(struct cfs_fs *cfs_fs, cfs_cred_t *cred,
 
 	RC_WRAP_LABEL(rc, out, cfs_access_check, cred, stat, CFS_ACCESS_WRITE);
 
-	ssize_t bs = dstore_get_bsize(dstore, &oid);
```

Comment:
Question: If we set the bsize while creating FS either default or user defined, should we remove the dstreo_get_bsize API and do cleanup? It will make sure no where mistakenly in future we use this API to get bsize, it may lead to false positive.